cernopendata/modules/theme/assets/semantic-ui/js/search/components/ListItemLabels.js

```diff
@@ -1,6 +1,28 @@
+import PropTypes from "prop-types";
 import React from "react";
 import { Label } from "semantic-ui-react";
 
+const ExperimentLabel = ({ name, withKey }) => {
```

Comment:
We can create a separate file `ExperimentLabel` and import it here!

Reply:
It's only being used in this file and I don't think it'll be used elsewhere anytime soon, so I think we can leave it for now, what do you think?

Reply:
okay, we can leave it here for now 👍